assets/lang/sklmessages_en_US.properties

```diff
@@ -40,7 +40,7 @@ game.output.type.text=Text
 
 game.conflict.title=Duplicate instance warning
```

Comment:
I'm not sure, but it seems to me that this sentence should be capitalized.

---
By the way, are `settings.lang.help` and `settings.lang.link` still needed in this file?